addons/hitreactions/XEH_postInit.sqf

```diff
@@ -0,0 +1,50 @@
+#include "script_component.hpp"
+
+if (GVAR(weaponDropChanceGunHit) + GVAR(weaponDropChanceArmHit) == 0) exitWith {};
```

Comment:
Either you need to add the `_needRestart` parameter (see https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#arguments-of-cba_fnc_addsetting) to the settings or this check has to go, as it could be changed mid-mission by the server.

Personally, I prefer if you don't have to restart the mission just to change a CBA setting, so I'd like to see the check removed.

Reply:
Holdover from early exit, removed the check